### PR TITLE
austin.intro.2.sanborn.jaclyn

### PR DESCRIPTION
**Pull request by sanborn08**

### CODE REVIEW

File: 02LessonTwo/homework/index.html [hunk 1]
```diff
@@ -1,10 +1,72 @@
...
+        <td>Israel</td>
+        <td>Vietnam</td>
+      </tr>
+    </table>
+  </div>
+
+  <div id=contact>
+    <p>
+      <br>
+        <a href="https://www.facebook.com/"><img src="http://findicons.com/files/icons/816/icons_web_2_0/128/facebook.png"></a>
```
Comment (fasteddiegarcia) [inline 02LessonTwo/homework/index.html:64]: Great work with tables and CSS. I'd like the social media links to go directly to your page, not fb landing page. Keep it up! i'm excited to see what you build next.

File: 03LessonThree/in-class-project/daily-planet.html [hunk 2]
```diff
@@ -1,10 +1,71 @@
...
+                        <a href="https://www.twitter.com/"><img src="http://findicons.com/files/icons/2137/twitter/128/twitter_11.png"></a>
+                        <a href="https://www.instagram.com/"><img src="http://findicons.com/files/icons/2745/sugar/128/instagram.png" ?></a>
+                    </td>
+                </tr>
+            </table>
+    </main>
+    </footer>
+</body>
+
+</html>
```
Comment (ethanstenis) [inline 03LessonThree/in-class-project/daily-planet.html:71]: Fantastic! The email link is clever. Next time, link to real social media accounts, if you want.

File: 04LessonFour/in-class-project/css/style.css [hunk 3]
```diff
@@ -1,84 +1,37 @@
...
+h1, h2, h3 {
+  font-weight:normal;
+}
+
+nav a {
+  color:white;
+}
+
+img, .header, .support {
+  border-radius:10px;
```
Comment (ethanstenis) [inline 04LessonFour/in-class-project/css/style.css:36]: Love the border radius!

File: 07LessonSeven/homework/css/style.css [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+nav {
+  background-color:black;
+  color:white;
+  width:100%;
+  height:90px;
+  margin-bottom:15px;
+  word-spacing:15px;
+  padding-left:140px;
```
Comment (ethanstenis) [inline 07LessonSeven/homework/css/style.css:12]: in order to get your navigation links to vertical center in your nav, add: vertical-align: middle; to 'nav'
You will also need to target "nav a" with "line-height: 27px;" or something about that size.

File: 10LessonTen/Checkpoint2/index.html [hunk 5]
```diff
@@ -2,9 +2,95 @@
...
+      <a href="http://www.skilledup.com/articles/life-front-end-web-developer-infographic"><img src="http://cdn.skilledup.com/wp-content/uploads/2014/10/Life-Front-End-Developer-Feature_1290x688_KL-940x501.jpg" alt="" /></a>
+    </div>
+  </section>
+  <section id="contact">
+    <div class=end>
+      <h2>Contact Me</h2>
+      <a href="https://twitter.com/slo_jac"><img src="http://wiredpen.com/wp-content/uploads/2016/05/Twitter-icon-horizontal.jpg" alt="Twitter" /></a>
+      <a href="https://www.facebook.com/jaclyn.sanborn.1"><img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTs-PukXt86dEkuNzoQ-RLvVtliIKV-PLRMV3FDlEjDJIh8owqa" alt="Facebook" /></a>
+    </div>
+  </section>
```
Comment (ethanstenis) [inline 10LessonTen/Checkpoint2/index.html:94]: nice clean design and layout. Well done. Did you play soccer in college?

File: 08LessonEight/in-class-project/exercise-06.html [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+</head>
+
+<body>
+    <header>
+        <h1>Recent From Blog</h1>
+        <p>Bacon ipsum dolor amet beef brisket ball tip, alcatra cupim meatball chicken hamburger tenderloin doner porchetta.</p>
+    </header>
+    <main class="clearfix">
+        <article class="box left">
+            <div class="keyboardimage">
```
Comment (ethanstenis) [inline 08LessonEight/in-class-project/exercise-06.html:16]: For some reason, this Keyboard.jpg image is broken. Does the src match the file name?

File: 12LessonTwelve/homework/index.html [hunk 7]
```diff
@@ -1,10 +1,77 @@
...
+                </div>
+            </div>
+        </div>
+    </section>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
 
 </body>
+
 </html>
```
Comment (ethanstenis) [inline 12LessonTwelve/homework/index.html:77]: Looks good. Just one small change to the pricing box when the media queries change.
screen shot 2016-09-13 at 5 16 16 pm
screen shot 2016-09-13 at 5 16 26 pm

**Pull request closed without merging**